pkg/datarepair/checker/checker_test.go

```diff
@@ -205,106 +202,3 @@ func makePointer(t *testing.T, planet *testplanet.Planet, pieceID string, create
 	err := pointerdb.Put(ctx, pieceID, pointer)
 	require.NoError(t, err)
 }
-
-func TestCheckerResume(t *testing.T) {
```

Comment:
curious why this resume test case is removed?

Reply:
the metainfo iteration part of the checker loop is no longer something that needs to be tested, since that iteration has been made into a separate service